desktop/cmp/store/StoreFilterField.js

```diff
@@ -25,11 +25,20 @@ export class StoreFilterField extends Component {
         /** Store to filter */
         store: PT.instanceOf(BaseStore).isRequired,
         /** Names of fields in store's records to filter by */
-        fields: PT.arrayOf(PT.string).isRequired
+        fields: PT.arrayOf(PT.string).isRequired,
+        /** Delay (in ms) used to buffer filtering of the store after the value changes from user input (default is 200ms). Set to 0 to filter immediately after user input. */
+        filterBuffer: PT.number
     };
 
     @setter @observable value = '';
 
+    constructor(props) {
+        super(props);
+
+        const {filterBuffer = 200} = this.props;
+        this._debouncedFilter = debounce(this.runFilter, filterBuffer);
+    }
```

Comment:
This kind of brings up the question I have long had about how to distiguish beween "fixed" props and "dynamic" props.  In this case,  this becomes a "fixed" prop.  Changing it after the first render of this component will have no effect.

Reply:
(Not implying that there is a problem with that)